src/Dotnet.Script/Program.cs

```diff
@@ -60,7 +63,7 @@ private static int Wain(string[] args)
                 {
                     if (!string.IsNullOrWhiteSpace(code.Value))
                     {
-                        await RunCode(code.Value, config.HasValue() ? config.Value() : "Release", debugMode.HasValue(), app.RemainingArguments, cwd.Value());
+                        await RunCode(code.Value, config.HasValue() ? config.Value() : "Release", debugMode.HasValue(), app.RemainingArguments.Concat(argsAfterDoubleHypen), cwd.Value());
```

Comment:
how did it work before? I mean, even the build script uses this https://github.com/filipw/dotnet-script/blob/master/build/Build.csx#L8

Reply:
ugh I is it because the CMD script wraps everything into a quote automatically?
I believe everything works fine when going through the DLL manually, but not when going through CMD script